src/displayapp/screens/StopWatch.cpp

```diff
@@ -146,7 +147,13 @@ void StopWatch::Refresh() {
     laps[lapsDone] = oldTimeElapsed + xTaskGetTickCount() - startTime;
 
     TimeSeparated_t currentTimeSeparated = convertTicksToTimeSegments(laps[lapsDone]);
-    lv_label_set_text_fmt(time, "%02d:%02d", currentTimeSeparated.mins, currentTimeSeparated.secs);
+    if (currentTimeSeparated.hours == 0) {
+      lv_label_set_text_fmt(time, "%02d:%02d", currentTimeSeparated.mins, currentTimeSeparated.secs);
+    } else {
+      lv_label_set_text_fmt(time, "%02d:%02d:%02d", currentTimeSeparated.hours, currentTimeSeparated.mins, currentTimeSeparated.secs);
+      lv_obj_set_style_local_text_font(time, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &jetbrains_mono_42);
+      lv_obj_align(time, msecTime, LV_ALIGN_OUT_TOP_MID, 0, 0);
```

Comment:
Because these are in the Refresh function, they are called constantly. It would be nice to find a solution to this. A bool or the DirtyValue template class will probably do the trick.

Reply:
I implemented it with a simple bool. Tried to name it my best :sweat_smile: